handbook/engineering/distribution/goals.md

```diff
@@ -2,41 +2,83 @@
 
 Goals are continuously updated and reviewed. If you find these goals do not reflect our current priorities or are out of date, please update them as soon as possible or add it as a topic to our [weekly sync](recurring_processes.md#weekly-distribution-team-sync).
 
-## Medium-term goals
+## Medium-term goals (~3-6 Months)
 
 ### Any engineer at Sourcegraph can create a release for all of our supported deployment types by running a single command
```

Comment:
Should we create a project to link here? (the status below indicates this is already in progress)

Reply:
I did not add one to the goal as we had multiple projects that contributed to this goal and I linked those to the milestones.